src/metrics.rs

```diff
@@ -74,16 +74,16 @@ impl<'a> Metric for CountMetric<'a> {
     // my_count:-1|c
     fn metric_type_format(&self) -> String {
         match *self {
-            CountMetric::Incr(stat) => {
-                let mut buf = String::with_capacity(3 + stat.len() + 4);
+            CountMetric::Incr(stat, amount) => {
+                let mut buf = String::with_capacity(3 + stat.len() + amount.to_string().len() + 4);
```

Comment:
Oops, didn't add this comment to the review:

I think this `4` we're adding to the capacity can drop down to a `3`, now that we're including the length of the amount. Same thing with the `5` in the Decr case below

Reply:
Alright updated to be 3/4 respectively!